### PR TITLE
Add test to Change password functionality

### PR DESCRIPTION
**Pull request by TetianaPerinha**
Task Name:
IL-308: Create auto-tests for functionality "Change password"  through a reset, Auth module.

Developer:
Tetiana Perinha

Changes:
Develop an automated test(happy path) to cover the "Change password"  through a reset, Auth module. Create api method to send a request to reset the password.

### CODE REVIEW

**Review by annstriganova — CHANGES_REQUESTED** [review]
I'm getting the following error

```python
(venv) astriganova@annsmac Iced-Latte-QA % python -m pytest ./tests
ImportError while loading conftest '/Users/astriganova/Desktop/Projects/Iced-Latte-QA/tests/conftest.py'.
tests/conftest.py:12: in <module>
    from configs import (
E   ImportError: cannot import name 'EMAIL_LOCAL_PART' from 'configs' (/Users/astriganova/Desktop/Projects/Iced-Latte-QA/configs.py)
```

**Comment by TetianaPerinha** [comment]
> I'm getting the following error
> 
> ```python
> (venv) astriganova@annsmac Iced-Latte-QA % python -m pytest ./tests
> ImportError while loading conftest '/Users/astriganova/Desktop/Projects/Iced-Latte-QA/tests/conftest.py'.
> tests/conftest.py:12: in <module>
>     from configs import (
> E   ImportError: cannot import name 'EMAIL_LOCAL_PART' from 'configs' (/Users/astriganova/Desktop/Projects/Iced-Latte-QA/configs.py)
> ```

**Comment by TetianaPerinha** [comment]
I added configs.py file

File: configs.py [hunk 1]
```diff
@@ -0,0 +1,45 @@
+# HOST = 'https://dev.api.it-sl.ru'
+
+
+HOST = "http://localhost:8083"
+# HOST = 'https://iced-latte.uk/backend'
+HOST_DB = "192.168.1.66"
```
Comment (annstriganova) [inline configs.py]: This seem to be your inner ip, that will be different for others. Is it possible to use `localhost` there?

File: configs.py [hunk 2]
```diff
@@ -0,0 +1,45 @@
...
+# HOST_DB = '172.20.10.3'
+# HOST_DB = 'localhost'
+
+PORT_DB = "5432"
+DB_NAME = "testdb"
+DB_USER = "postgres"
+DB_PASS = "postgres"
+
+# data for creating user#1
+email = "icedlate.test@gmail.com"
```
Comment (annstriganova) [inline configs.py:10]: Is it a real mail box?

File: configs.py [hunk 3]
```diff
@@ -0,0 +1,45 @@
+# HOST = 'https://dev.api.it-sl.ru'
```
Comment (annstriganova) [inline configs.py]: This commented out string should be removed 😄

**Review by annstriganova — CHANGES_REQUESTED** [review]
Another issue:

```python
(venv) astriganova@annsmac Iced-Latte-QA % python -m pytest ./tests
ERROR: usage: __main__.py [options] [file_or_dir] [file_or_dir] [...]
__main__.py: error: unrecognized arguments: --reruns --reruns-delay 5 ./tests
  inifile: /Users/astriganova/Desktop/Projects/Iced-Latte-QA/pytest.ini
  rootdir: /Users/astriganova/Desktop/Projects/Iced-Latte-QA
```

File: configs.py (hunk 3 above)
Comment (TetianaPerinha) [reply configs.py]: Resolve

File: configs.py (hunk 1 above)
Comment (TetianaPerinha) [reply configs.py]: Resolved

**Comment by TetianaPerinha** [comment]
> Another issue:
> 
> ```python
> (venv) astriganova@annsmac Iced-Latte-QA % python -m pytest ./tests
> ERROR: usage: __main__.py [options] [file_or_dir] [file_or_dir] [...]
> __main__.py: error: unrecognized arguments: --reruns --reruns-delay 5 ./tests
>   inifile: /Users/astriganova/Desktop/Projects/Iced-Latte-QA/pytest.ini
>   rootdir: /Users/astriganova/Desktop/Projects/Iced-Latte-QA
> ```

I added to requirements.txt  plug-in  pytest-rerunfailures==14.0. Install requirements.txt please , it should resolve the problem

File: configs.py (hunk 2 above)
Comment (TetianaPerinha) [reply configs.py:10]: yes, I created it for testing purpose

**Review by annstriganova — APPROVED** [review]
Approved.

**Review by Sunagatov — APPROVED** [review]

**Comment by ilsinyakov** [comment]
Approved

**Pull request merged**